### PR TITLE
Fix survey template picker

### PR DESCRIPTION
**Pull request by eikhr**
# Description

This was very buggy. It was calling a function `destroy()` that didn't exist, that I couldn't even find it in old code, so idk how that has ever worked. The template picker dropdown also has a hidden button, that it was still possible to hit, causing the dropdown to be toggled twice and instantly close again. This is a super hacky workaround and should definitely be refactored later on, but for now I set the height and width to 0 so that it is impossible to hit.

# Result

It is actually possible to use the survey templates again.

# Testing

- [x] I have thoroughly tested my changes.

Seems to work locally:)
---

Resolves ABA-465

### CODE REVIEW

**Review by juniwbjerde — APPROVED** [review]
LGTM!

File: app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx [hunk 1]
```diff
@@ -64,21 +60,22 @@ function TemplateTypeDropdownItems({
...
-              e.stopPropagation();
-              destroy();
-              push(link(eventType));
-            }}
-          >
-            {eventTypeToString(eventType)}
-          </Link>
-        </Dropdown.ListItem>
-      ))}
+      {(Object.keys(EVENT_CONSTANTS) as (keyof typeof EVENT_CONSTANTS)[]).map(
```
Comment (ivarnakken) [inline app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx:63]:
```suggestion
      {(Object.keys(EVENT_CONSTANTS) as EventType[]).map(
```
Is this not the same?

File: app/routes/surveys/components/surveys.css [hunk 2]
```diff
@@ -191,8 +191,11 @@
 }
 
 .templateDropdown {
-  position: absolute;
   color: transparent;
+  position: absolute;
+  bottom: 0;
+  height: 0;
+  width: 0;
```
Comment (ivarnakken) [inline app/routes/surveys/components/surveys.css:198]: Wouldn't hurt with a comment but okay

**Review by ivarnakken — APPROVED** [review]
Nice! 💯 Much cleaner!

Works fine for me btw

File: app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx (hunk 1 above)
Comment (eikhr) [reply app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx:63]: They should be equal, but it wouldn't actually check that the keys are the same strings as EventType

File: app/routes/surveys/components/surveys.css (hunk 2 above)
Comment (eikhr) [reply app/routes/surveys/components/surveys.css:198]: It wouldn't hurt, but I'm planning on refactoring it soon™️

**Pull request merged**